internal/policy/policy.go

```diff
@@ -229,9 +254,10 @@ func (p *policy) loadPolicy(ctx context.Context, policyRef string) error {
 		if err := yaml.Unmarshal([]byte(policyRef), &ecp); err == nil && ecp.APIVersion != "" {
 			p.EnterpriseContractPolicySpec = ecp.Spec
 		} else {
-			log.Debugf("Problem parsing EnterpriseContractPolicy from %q", policyRef)
+			log.Debugf("Unable to parse EnterpriseContractPolicy from %q", policyRef)
+			log.Debug("Attempting to parse as EnterpriseContractPolicySpec")
 			if err := yaml.Unmarshal([]byte(policyRef), &p.EnterpriseContractPolicySpec); err != nil {
-				log.Debugf("Problem parsing EnterpriseContractPolicySpec from %q", policyRef)
+				log.Debugf("Unable to parse EnterpriseContractPolicySpec from %q", policyRef)
```

Comment:
NIT: It would be good to make this message different than the one above so we can more easily tell where it comes from.

Reply:
This is different from the above. It accurately describes that it cannot parse `EnterpriseContractPolicySpec` whereas the message above describes that it cannot parse `EnterpriseContract Policy`